topologies/datacenter-latest/labguides/source/bgp-with-cvp.rst

```diff
@@ -0,0 +1,170 @@
+Layer 3 Leaf-Spine
+==================
+
+.. image:: images/cvp-l3ls/l3ls_1.png
+   :align: center
+   :width: 50 %
+
+.. note:: Did you know the “bgp” script is composed of Python code that
+          uses the CloudVision Portal REST API to automate the provisioning of
+          CVP Configlets. The configlets that are configured via the REST API
+          are ``Spine1-BGP-Lab``, ``Spine2-BGP-Lab``, ``Leaf1-BGP-Lab``,
+          ``Leaf2-BGP-Lab``, ``Leaf3-BGP-Lab``, ``Leaf4-BGP-Lab``.
+
+.. note:: The manually-entered commands below that are part of this lab are
```

Comment:
@jrecchia1029 Remove this note, it is not needed since it is referencing the login.py script when a user would ssh into the topology to configure the lab.